h3/src/frame.rs

```diff
@@ -12,26 +13,21 @@ use crate::{
         frame::{self, Frame, PayloadLen},
         stream::StreamId,
     },
-    quic::{RecvStream, SendStream},
+    quic::{BidiStream, RecvStream, SendStream},
     stream::WriteBuf,
 };
 
-pub struct FrameStream<S>
-where
-    S: RecvStream,
-{
+pub struct FrameStream<S, B> {
     stream: S,
     bufs: BufList<Bytes>,
     decoder: FrameDecoder,
     remaining_data: usize,
     /// Set to true when `stream` reaches the end.
     is_eos: bool,
+    _phantom_buffer: PhantomData<B>,
```

Comment:
Was this marker required? How come? I imagine probably yes, but I'm struggling to catch exactly where/why...

Reply:
```
impl<S, B> FrameStream<S, B>
where
    S: BidiStream<B>,
    B: Buf,
{
```

If FrameStream doesn't have the B parameter, this gives an error that B is unconstrained...

Reply:
#94 would solve this